osbs/build/plugins_configuration.py

```diff
@@ -363,6 +363,15 @@ def render_inject_parent_image(self):
 
         self.pt.set_plugin_arg(phase, plugin, 'koji_parent_build', koji_parent_build)
 
+    def render_koji_import(self, use_auth=None):
+        if not self.pt.has_plugin_conf('exit_plugins', 'koji_import'):
+            return
+
+        koji_target = self.user_params.koji_target.value
+        if not self.pt.set_plugin_arg_valid('exit_plugins', 'koji_import', 'target', koji_target):
```

Comment:
Looking at `koji_import` plugin, I don't think `target` parameter is used at all.

Reply:
Fixed, or at least, I'm not doing anything to render koji_import anymore.